elasticsearch/linux_distributions/opendistro-tar-install.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Keep this change. meaning remove the year

Reply:
I can re-make that change later - I want this to be a simple revert.